### ISSUE TITLE
Check installed conda-smithy against current feedstock conda-smithy

### ISSUE THREAD

**Issue by pmlandwehr** [pmlandwehr/tick-feedstocks#9]
Is there metadata for this?

**Comment by jakirkham** [comment]
Not really. That's sort of intentional so as to avoid re-renderings that only update this metadata. Though we have tried to standardize commit messages from re-renderings. Maybe doing some `git` log parsing would solve this. Would recommend `gitpython` to make this easier.

**Comment by jakirkham** [comment]
OTOH one can just always run the re-render. Is there some reason not to?

**Comment by pmlandwehr** [comment]
No, not at all. It just seems ungainly to have the program update the version and then inadvertently revert the regenerated code.
